static/js/containers/ProfileFormContainer.js

```diff
@@ -81,78 +82,13 @@ class ProfileFormContainer extends React.Component {
     }
     dispatch(updateProfile(username, profile));
     this.updateProfileValidation(this.props, profile, validator);
-  }
-
-  setProfileStep: Function = (step: string): void => {
-    const { dispatch } = this.props;
-    dispatch(setProfileStep(step));
-  };
-
-  setDeletionIndex: Function = (index: number): void => {
-    const { dispatch } = this.props;
-    dispatch(setDeletionIndex(index));
-  }
-
-  setShowEducationDeleteDialog: Function = (bool: boolean): void => {
-    const { dispatch } = this.props;
-    dispatch(setShowEducationDeleteDialog(bool));
-  }
-
-  setShowWorkDeleteDialog: Function = (bool: boolean): void => {
-    const { dispatch } = this.props;
-    dispatch(setShowWorkDeleteDialog(bool));
-  }
-
-  setShowWorkDeleteAllDialog: Function = (bool: boolean): void => {
-    const { dispatch } = this.props;
-    dispatch(setShowWorkDeleteAllDialog(bool));
-  };
-
-  setShowEducationDeleteAllDialog: Function = (bool: boolean): void => {
-    const { dispatch } = this.props;
-    dispatch(setShowEducationDeleteAllDialog(bool));
-  };
-
-  setUserPageDialogVisibility: Function = (bool: boolean): void => {
-    const { dispatch } = this.props;
-    dispatch(setUserPageDialogVisibility(bool));
   };
 
   setWorkHistoryEdit: AsyncActionHelper = (bool: boolean) => {
     const { dispatch } = this.props;
     return dispatch(setWorkHistoryEdit(bool));
   }
```

Comment:
should be pretty easy to follow that createSimpleActionHelpers pattern with the `AsyncActionHelper` type functions, no?